apps/backend/src/libs/modules/cloud-formation/libs/enums/enums.ts

```diff
@@ -0,0 +1,5 @@
+export { Capability } from "./cloudformation-capability.enum.js";
```

Comment:
```suggestion
export { Capability } from "./capability.enum.js";
```
Same for all. Since all files are inside cloud-formation folder, we don't need to specify it again